torch/csrc/jit/python/script_init.cpp

```diff
@@ -1183,7 +1183,7 @@ void initJitScriptBindings(PyObject* module) {
             m.register_attribute(name, type, toIValue(value, type));
           })
       .def(
-          "_create_method_from_trace",
+          "_create_method_from_trace_with_tuple",
```

Comment:
Could we leave this with its old name? Even though this method is prefixed with an underscore, I would not be surprised if there are users who use this api directly...

Reply:
OK, this old name could be kept.(Pls. I think this is the same issue about if we should maintain the backward compatibility for a non-standard behavior right?)